src/engraving/rw/read302/read302.cpp

```diff
@@ -316,6 +452,19 @@ void Read302::fixInstrumentId(Instrument* instrument)
         id = u"marching-cymbals";
     } else if (id == u"bass-drum" && trackName == u"bass drums") {
         id = u"marching-bass-drums";
+    } else if (id.startsWith(u"mdl-")) {
+        // See https://github.com/musescore/mdl/blob/master/resources/instruments/mdl_1_3_0.xml
+        if (id == u"mdl-snareline" || id == u"mdl-snareline-a" || id == u"mdl-snaresolo" || id == u"mdl-snaresolo-a") {
+            id = u"marching-snare";
+        } else if (id == u"mdl-tenorline" || id == u"mdl-tenorsolo" || id == u"mdl-flubs") {
+            id = u"marching-tenor-drums";
+        } else if (id == u"mdl-bassline-10" || id == u"mdl-bassline-5") {
+            id = u"marching-bass-drums";
+        } else if (id == u"mdl-cymballine") {
+            id = u"marching-cymbals";
+        } else if (id == u"mdl-showtenorline" || id == u"mdl-rail" || id == u"mdl-drumset" || id == u"mdl-sampler") {
+            id = u"drumset";
```

Comment:
@zacjansheski, instruments created in MS 3.6 do have IDs. Here I convert the MDL IDs into corresponding instruments from our [instruments spreadsheet] and [`instruments.xml`]. Same considerations apply as above. Let me know if you disagree with any of my choices, or would like to the MDL instruments to be added to `instruments.xml`.

[instruments spreadsheet]: https://docs.google.com/spreadsheets/d/1SwqZb8lq5rfv5regPSA10drWjUAoi65EuMoYtG-4k5s/edit?gid=516529997#gid=516529997
[`instruments.xml`]: https://github.com/musescore/MuseScore/blob/master/share/instruments/instruments.xml